src/components/capture/creation/index.tsx

```diff
@@ -122,7 +122,7 @@ function NewCaptureModal() {
                 setFormSubmitting(true);
                 setFormSubmitError(null);
                 setActiveStep(Steps.WAITING_FOR_DISCOVER);
-                axios
+                axiosInstance
```

Comment:
This change is all over... we no longer want the default axios we want to make sure we fetch the one with our settings.